.circleci/cluster/manager1_config.yaml

```diff
@@ -50,5 +50,8 @@ ssl_inputs:
   external_key_path: /etc/cloudify/external_key.pem
   external_ca_cert_path: /etc/cloudify/ca.pem
 
+flask_security:
+  encoding_alphabet: 5TNRm2piPIOXn6FVAhcJeUSjswEHYbg
```

Comment:
Why are we forcing this to one value for CI?

Reply:
Just for keeping it simple and stable

Reply:
We've actually managed to cause ourselves a variety of hidden bugs by having values that don't change in images before, so it'd be better if we didn't do this if we can avoid it.